### DISCUSSION TITLE
How to run picom with --experimental-backends in i3?

### DISCUSSION THREAD

**Discussion by EthanRodrigo** [i3/i3#4756] [Q&A]
Hello guys. I'm using i3 with picom as my compositor. And I needed to use the blur background feature so I changed the `picom.conf` and run in the terminal with `picom --experimental-backend` and it worked. But when I try to add that into i3 with `exec_always --no-startup-id picom --experimental-backend` It's not working. Then I added that line into a bash file and try to run it with i3 and it's also not working. Anyone know how to fix this?

**Comment by Airblader** [comment]
For starters, you have a typo in the command.

**Reply by EthanRodrigo** [reply]
edited

**Reply by Airblader** [reply]
When you say "it's not working", how do you determine that? What exactly did you do after editing the config, what did you expect to see and what did you see instead?

**Reply by EthanRodrigo** [reply]
I need the background of an app to be blured. In a nutshell I can't run a `picom` instance that has `--experimental-backends`

**Reply by orestisfl** [reply]
Have you tried running it from the terminal and see if there's any error message?

**Comment by AllenVenus** [comment]
for anyone in the future here what worked for me:
`picom --experimental-backends -b`

**Comment by carafizi1** [comment]
Have been using this since early 2021, since the latest update (2022/11/17) it stopped working.

this command:
`picom --experimental-backends -b
`
spits out this error:
`picom: unrecognized option '--experimental-backends' 
picom (vgit-5d150)
`
My solution was removing the parameter, not sure if it'll break any of my configs but at least i got transparency and some effects back to work:

my .i3/config:
`exec_always --no-startup-id picom -b`

**Reply by EthanRodrigo** [reply]
Yes since the new update, there's no need of specifying `--experimentatl-backends` option. I use `--config` option though.
